content/5_Plat/Bitsets.mdx

```diff
@@ -74,6 +77,55 @@ void init() {
 }
 ```
 
+</CPPSection>
+<PySection>
+
+```py
+# BeginCodeSnip{Disjoint Set Union}
+class DisjointSet:
+	parent = {}
+	sz = []
+
+	def __init__(self, sz) -> int:  # Construct the DSU
```

Comment:
you still need type annotations for the arguments

Reply:
Done !